iotedgehubdev/edgemanager.py

```diff
@@ -193,6 +188,28 @@ def start_solution(self, deployment_config, verbose):
             cmd_up = ['docker-compose', '-f', EdgeManager.COMPOSE_FILE, 'up', '-d']
         Utils.exe_proc(cmd_up)
 
+    @staticmethod
+    def loginRegistries(module_content):
+        registryCredentials = None
+        if module_content['$edgeAgent']['properties.desired']['runtime']:
+            settings = module_content['$edgeAgent']['properties.desired']['runtime']['settings']
+            if settings:
+                registryCredentials = settings['registryCredentials']
```

Comment:
NIT: you can use one line `registryCredentials  = module_content.get('$edgeAgent', {}).get('properties.desired', {}).get('runtime', {}).get('settings', {}).get('registryCredentials', {})`

Reply:
Thanks for the suggestion, will update.